inc/template-tags.php

```diff
@@ -344,6 +345,7 @@ function responsive_primary_nav() {
 				'menu_id'        => 'primaryNav-menu',
 				'menu_class'     => 'primaryNav-menu',
 				'container_tag'  => 'ul',
```

Comment:
I don't think `container_tag` is a valid argument. `container` is tho, but that shouldn't be set to a `ul` element; instead it should use either a `div` or `nav` element i would think, since the `items_wrap` argument defaults to `ul` element.